tests/Feature/Http/Controllers/Auth/RegistrationTest.php

```diff
@@ -48,11 +48,11 @@ public function testNewUsersCanRegister(): void
         $response = $this->post($route, $formData);
 
         // Then
-        $user = User::where('email', '=', $email)->first();
+        $user = AuthorizedUser::where('email', '=', $email)->first();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::where('email', '=', $email)->first();
````